frontend/src/libs/components/scroll-to-top-button/libs/scroll-value.enum.ts

```diff
@@ -0,0 +1,6 @@
+const SCROLL_VALUE = {
+  TOP: 0,
+  SCROLLED_BELOW_200: 200,
```

Comment:
I am a bit confused with name SCROLLED_BELOW_200, is 200 some kind of a bound? Let's give it more meaningful name